CHANGELOG.md

```diff
@@ -5,6 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [3.6.4] - 2025-03-24
```

Comment:
This is missing the other two PRs that went into this milestone. See https://github.com/Automattic/Co-Authors-Plus/milestone/38?closed=1

Reply:
Added! Thank you.